lib/conjur/command/policy.rb

```diff
@@ -74,19 +75,19 @@ class Conjur::Command::Policy < Conjur::DSLCommand
         policy = Policy.new(api.role(id), api.resource(id))
 
         validate_retire_privileges(policy, options)
-        
+
```

Comment:
Well, the `retire` command should come from `conjur-asset-policy` since it describes `load` as well. Having parts of the workflow in different projects is confusing.